packages/contracts-bedrock/STYLE_GUIDE.md

```diff
@@ -60,9 +71,12 @@ Unless explicitly discussed otherwise, you MUST include the following basic upgr
 
 ### Versioning
 
-All (non-library and non-abstract) contracts MUST extend the `Semver` base contract which exposes a `version()` function that returns a semver-compliant version string.
-During the Bedrock development process the `Semver` value for all contracts SHOULD return `0.0.1` (this is not particularly important, but it's an easy standard to follow).
-When the initial Bedrock upgrade is released, the `Semver` value MUST be updated to `1.0.0`.
+All (non-library and non-abstract) contracts MUST extend the `Semver` base contract which
+exposes a `version()` function that returns a semver-compliant version string.
+
+Contracts with a `Semver` value of less than `1.0.0` were pre-bedrock upgrade. When the
```

Comment:
Perhaps we can generalize this by saying something along the lines of "contracts pre `1.0.0` are not production ready"